desk/lib/dm.hoon

```diff
@@ -148,204 +171,70 @@
     =/  reply-seal  [id parent-id now ~]
     ?:  (~(has by dex.pac) id)  [pac replies]
     =.  dex.pac  (~(put by dex.pac) id now)
-    [pac (put:on:replies:c replies now reply-seal memo.delta)]
+    [pac (put:on:replies:c replies now %& reply-seal memo.delta)]
   ::
       %del
     =/  tim=(unit time)  (~(get by dex.pac) id)
     ?~  tim  [pac replies]
-    =/  =time  (need tim)
-    =^  reply=(unit reply:c)  replies
-      (del:on:replies:c replies time)
-    =.  dex.pac  (~(del by dex.pac) id)
-    [pac replies]
```

Comment:
What happened to the `=.  dex.pac` here? The new code just produces `pac` as-is, always, but it should delete the lookup for this reply right?